ci/common/build.bash

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env bash
+
+# Copyright (c) 2018-2020 NVIDIA Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Released under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+
+################################################################################
+# Thrust and CUB build script for gpuCI
+################################################################################
+
+set -e
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> ${@}\n"
+}
+
+################################################################################
+# VARIABLES - Set up bash and environmental variables.
+################################################################################
+
+# Get the variables the Docker container set up for us: ${CXX}, ${CUDACXX}, etc.
+source /etc/cccl.bashrc
+
+# Set path and build parallel level
+export PATH=/usr/local/cuda/bin:${PATH}
+
+# Set home to the job's workspace.
+export HOME=${WORKSPACE}
+
+# Switch to project root; also root of repo checkout.
+cd ${WORKSPACE}
+
+# If it's a nightly build, append current YYMMDD to version.
+if [ "${BUILD_MODE}" == "branch" ]; then
+  export VERSION_SUFFIX=`date +%y%m%d`
+fi
+
+# The Docker image sets up `${CXX}` and `${CUDACXX}`.
+CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_CUDA_COMPILER=${CUDACXX}"
+
+# If it's a nightly build, build all configurations.
+if [ "${BUILD_MODE}" == "branch" ]; then
+  CMAKE_FLAGS="${CMAKE_FLAGS} -DTHRUST_MULTICONFIG_WORKLOAD=FULL"
+fi
+
+CTEST_FLAGS=""
+
+if [ "${BUILD_KIND}" == "cpu" ]; then
```

Comment:
There is a gpuCI environment variable for this called `BUILD_TYPE`. Check out all of the available variables here: https://docs.rapids.ai/maintainers/gpuci/#environment-variables

Reply:
Ah cool thanks.